satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -631,2088 +684,2272 @@ datasets:
     extract_byte: 17
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_cmrt3:
     name: cloud_test_cmrt3
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_test_result
-    long_name: cloud_mask_test_cmrt3
     extract_byte: 18
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_cmrt4:
     name: cloud_test_cmrt4
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_test_result
-    long_name: cloud_mask_test_cmrt4
     extract_byte: 19
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_cmrt5:
     name: cloud_test_cmrt5
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_test_result
-    long_name: cloud_mask_test_cmrt5
     extract_byte: 20
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_dust:
     name: cloud_test_dust
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_test_result
-    long_name: cloud_mask_test_dust
     extract_byte: 21
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_ash:
     name: cloud_test_ash
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_test_result
-    long_name: cloud_mask_test_ash
     extract_byte: 22
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_dust_ash:
     name: cloud_test_dust_ash
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_test_result
-    long_name: cloud_mask_test_dust_ash
     extract_byte: 23
+    flag_values: [0,1]
+    flag_meanings: ['Cloud undetected','Cloud detected']
+    standard_name: status_flag
 
   cloud_test_cmrt6:
     name: cloud_test_cmrt6
     resolution: 2000
     file_type: nc_fci_test_clm
     file_key: cloud_mask_cmrt6_test_result
-    long_name: cloud_mask_cmrt6_result
+    fill_value: -127
+    standard_name: status_flag
+    import_enum_information: True
 
   product_quality_clmtest:
     name: product_quality_clmtest
     file_type: nc_fci_test_clm
     file_key: product_quality
-    long_name: product_quality_index
+    standard_name: product_quality
 
   product_completeness_clmtest:
     name: product_completeness_clmtest
     file_type: nc_fci_test_clm
     file_key: product_completeness
-    long_name: product_completeness_index
+    standard_name: product_completeness
 
   product_timeliness_clmtest:
     name: product_timeliness_clmtest
     file_type: nc_fci_test_clm
     file_key: product_timeliness
-    long_name: product_timeliness_index
-
+    standard_name: product_timeliness
 
   # ASR
```

Comment:
Some general comments on the ASR entries:
- It seems like the datasts have changed, which should not be the case, we should still include the same datasets as before, just with adjusted/added `long_name`, `standard_name` and `cell_method`
- Please use the same `long_name` as you used for the SEVIRI BUFR reader (e.g. `TOA Brightness Temperature at 8.7um (segment mean over all pixels)`)
- Shouldn't the standard names be `toa_brightness_temperature`, `toa_bidirectional_reflectance` and `toa_outgoing_radiance` for brightness temperatures, reflectance and radiance, respectively.

All poit should be easily fixed by simply modifying the script used to generated the ASR entries.

Reply:
Changed the long names to be like the SEVERI ASR
Changed the standard names as suggested

Branch was updated

Reply:
There is still the issue that the list of dataset is different in this branch compared to `main` as far as I can see. Can you see if you can fix it, otherwise provide me with your version of the script used to generate the yaml entries.

Reply:
The issue is that the variable progression in the original file is not really logical (for example, the ordering between bt, radiance and reflectance changes), and that the python script I inherited didn't reproduce that, so I think the "non-sliced" variables like bt_min were manually added. My script does them automatically. The content I believe is the same in the end but then the order differs.

Reply:
Yeah, I think the non-sliced variables were indeed added manually. Anyway, now the order and more importantly the number datasets seems to be unchanged indeed :)